docs/source/quickstart.rst

```diff
@@ -84,7 +80,7 @@ To do this, you can either
 
 1. Install Chaco and its :ref:`dependencies` from `PyPI
    <http://pypi.python.org/pypi>`_ using `easy_install
-   <http://packages.python.org/distribute/easy_install.html>`_ (part of
```

Comment:
I've removed it. Also, should we keep the references to ETS or purge all of them and replace them with references to Canopy?